src/test/java/com/urswolfer/gerrit/client/rest/http/changes/ChangeApiRestClientTest.java

```diff
@@ -23,9 +23,11 @@
 import com.google.gerrit.extensions.client.ListChangesOption;
 import com.google.gerrit.extensions.common.*;
 import com.google.gson.JsonElement;
+import com.google.gson.JsonPrimitive;
 import com.urswolfer.gerrit.client.rest.http.GerritRestClient;
 import com.urswolfer.gerrit.client.rest.http.common.GerritRestClientBuilder;
 import org.easymock.EasyMock;
+import org.hamcrest.core.AllOf;
```

Comment:
replace with `import java.util.EnumSet;`

Reply:
Good catch, thanks. Fixing it now.

Reply:
Fixed.